components/experimental/tests/transliterate/lower_ascii.rs

```diff
@@ -0,0 +1,110 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use core::any::TypeId;
+
+use icu::casemap::CaseMapper;
+use icu_experimental::transliterate::{
+    provider::TransliteratorRulesV1Marker, CustomTransliterator, RuleCollection,
+    RuleCollectionProvider, Transliterator,
+};
+use icu_locale::LanguageIdentifier;
+use icu_provider::prelude::*;
+
+#[allow(unused_imports)]
+mod provider {
+    use icu_experimental;
+    include!("data/provider.rs");
+}
+
+struct TransliteratorMultiSourceProvider<'a>(
+    RuleCollectionProvider<'a, icu_properties::provider::Baked, icu_normalizer::provider::Baked>,
+);
+
+impl<'a, M> DataProvider<M> for TransliteratorMultiSourceProvider<'a>
+where
+    M: DataMarker,
+    RuleCollectionProvider<'a, icu_properties::provider::Baked, icu_normalizer::provider::Baked>:
+        DataProvider<M>,
+{
+    fn load(&self, req: DataRequest) -> Result<DataResponse<M>, DataError> {
+        println!("{:?} {req:?}", M::INFO);
+        if TypeId::of::<M>() == TypeId::of::<TransliteratorRulesV1Marker>() {
+            let mut silent_req = req;
+            silent_req.metadata.silent = true;
+            match DataProvider::<TransliteratorRulesV1Marker>::load(
+                &provider::TestingProvider,
+                silent_req,
+            ) {
+                Ok(response) => {
+                    return Ok(DataResponse {
+                        metadata: response.metadata,
+                        payload: response.payload.dynamic_cast()?,
+                    })
+                }
+                Err(DataError {
+                    kind: DataErrorKind::IdentifierNotFound,
+                    ..
+                }) => (),
+                Err(e) => return Err(e),
+            }
+        }
+        self.0.load(req)
+    }
+}
+
+#[derive(Debug)]
+struct LowercaseTransliterator(CaseMapper);
```

Comment:
suggestion: I suspect these should be exported by the transliterator crate

but we really need to figure out a better solution for bundling "default" transliterators anyway

Reply:
Yes, they should be, but I didn't feel like making big changes right now; I want to show how to do this with minimal patches against 1.5

Reply:
Yeah I'm not proposing that for this PR